types/address/hash.go

```diff
@@ -0,0 +1,47 @@
+package address
+
+// TODO remove this functions and use the latest SDK.
```

Comment:
should this file be removed?

Reply:
up

Reply:
I still need to check with @aaronc  about the ModuleID.